.devcontainer/Dockerfile

```diff
@@ -14,28 +14,28 @@
 
 FROM ghcr.io/eclipse-velocitas/devcontainer-base-images/python:latest
 
-# [Option] Install zsh
-ARG INSTALL_ZSH="true"
-# [Option] Upgrade OS packages to their latest versions
-ARG UPGRADE_PACKAGES="false"
-# [Option] Enable non-root Docker access in container
-ARG ENABLE_NONROOT_DOCKER="true"
-# [Option] Use the OSS Moby Engine instead of the licensed Docker Engine
-ARG USE_MOBY="true"
 # Force dapr to use localhost traffic
 ENV DAPR_HOST_IP="127.0.0.1"
-
-# Install needed packages and setup non-root user. Use a separate RUN statement to add your
-# own dependencies. A user of "automatic" attempts to reuse an user ID if one already exists.
-ARG USERNAME=automatic
-ARG USER_UID=1000
-ARG USER_GID=$USER_UID
-ENV DOCKER_BUILDKIT=1
-
 # Add daprd to the path for the VS Code Dapr extension.
 ENV PATH=$PATH:/home/vscode/.dapr/bin
 
+ENV DOCKER_BUILDKIT=1
+
 COPY scripts/*.sh /tmp/scripts/
 RUN find /tmp/scripts/ -type f -iname "*.sh" -exec chmod +x {} \;
 RUN /bin/bash /tmp/scripts/container-set.sh
 RUN /bin/bash /tmp/scripts/configure-proxies.sh
+
+# Fix for bug in devcontainer features
+ADD https://raw.githubusercontent.com/devcontainers/features/main/src/common-utils/install.sh /tmp/scripts/install-common.sh
+RUN UID="4000" bash /tmp/scripts/install-common.sh
+ADD https://raw.githubusercontent.com/devcontainers/features/main/src/docker-in-docker/install.sh /tmp/scripts/install-dind.sh
+RUN VERSION="20.10" bash /tmp/scripts/install-dind.sh
```

Comment:
do we have to make sure that we set the entire devContainer also to 20.10 ubuntu?
I believe that would be possible in the devContainer.json file.
I have to admit I am not sure... it just stood out to me

Reply:
it is not ubuntu 20.10 but docker 20.10 and no we don't have to change the devcontainer for that